### ISSUE TITLE
autogen.sh errors

### ISSUE THREAD

**Issue by greenwald** [bat/bat#51]
Should we try to remove these?

```
src/Makefile.am:65: patsubst %.h, $(top_srcdir: non-POSIX variable name
src/Makefile.am:65: (probably a GNU make extension)
test/Makefile.am:41: `LDFLAGS' is a user variable, you should not override it;
test/Makefile.am:41: use `AM_LDFLAGS' instead.
```

**Comment by oschulz** [comment]
If possible, sure. :-)
We don't need it for 1.0, though.

**Comment by fredRos** [comment]
I can have look at it but it only makes sense after #66 and #67 are in master

**Comment by oschulz** [comment]
I'm currently doing some changes to the whole build system, to use Automake for the examples and
to integrate the examples into "make (dist)check". This may clear up some of the autogen warnings as a side-effect.

**Comment by fredRos** [comment]
Fixed by #72
